ironfish-rust/src/transaction/mints.rs

```diff
@@ -63,6 +63,7 @@ impl MintBuilder {
         &self,
         spender_key: &SaplingKey,
         public_key_randomness: &Fr,
```

Comment:
I wonder if this is better than regenerating from the randomness as it was doing before. You can still regenerate as it was doing in line 78, but keep all the other updates below you made to the MintDescription/verify proof. Up to you

Reply:
yeah both would work. I figure since we store the rpk in transaction, I just pass in instead of regenerating individually in spend, output and mint since its repetitive computation.